src/lib/json-schema.base.ts

```diff
@@ -25,7 +24,7 @@ export interface ValidationResult {
  * Provides methods for validation and removal of extra properies not allowed in the schema
  */
 export class JSONSchema {
-  private _schema: JSONSchema7
+  private _schema: any
```

Comment:
This and the below types were changed to `any` to allow for additional properties in schemas that are not in the JSONSchema7 spec. Ajv is already ignoring them due to the `strictSchema: false` option above, but this ensures that the generated files satisfy the typescript compiler.

Reply:
can't we extrend the type instead?

Reply:
thanks for the explanation.